tools/smartos-release

```diff
@@ -75,34 +76,51 @@ fi
 
 set -o errexit
 
-# Note that ${BRANCH} appears only once here as we assume that for release
-# builds, all project branches and smartos-live itself use the same branch
-# name. See $(PUB_BRANCH_DESC) in the top-level Makefile for more details.
-SOURCE=/${MANTA_USER}/public/builds/platform/${BRANCH}-${TIMESTAMP}/platform/
 SMARTOS=/${MANTA_USER}/public/SmartOS
-DESTINATION=${SMARTOS}/${TIMESTAMP}
+SMARTOS_RELEASE=${SMARTOS}/${TIMESTAMP}
+BITS=output/bits/platform
 
 start_time=$(date +%s)
-echo "Creating release snaplinks under ${DESTINATION}"
 
-mmkdir -v -p ${DESTINATION}
-mfind ${SOURCE} -t o | while read OBJECT; do
-    mln ${OBJECT} ${DESTINATION}/$(basename ${OBJECT})
+echo "Uploading release objects to ${SMARTOS_RELEASE}"
+mmkdir -p ${SMARTOS_RELEASE}
+
+BUILD_FILES="platform-${BRANCH}-${TIMESTAMP}.tgz
+    platform-${BRANCH}-${TIMESTAMP}.imgmanifest
+    boot-${BRANCH}-${TIMESTAMP}.tgz
+    tests-${BRANCH}-${TIMESTAMP}.tgz
+    smartos-${TIMESTAMP}.iso
+    smartos-${TIMESTAMP}-USB.img.gz
+    smartos-${TIMESTAMP}.vmwarevm.tar.gz"
+
+for file in ${BUILD_FILES}; do
+    mput -f ${BITS}/$file ${SMARTOS_RELEASE}/$file
+done
+
+ADDITIONAL_FILES="SINGLE_USER_ROOT_PASSWORD.txt
+    build.log
+    changelog.txt
+    configure-projects
+    gitstatus.json
+    index.html
+    md5sums.txt"
+for file in ${ADDITIONAL_FILES}; do
```

Comment:
Sorry to be nitty, but I don't see the point in two loops or even two separate variables?

Reply:
That's ok - it was really a vague attempt to make the script a bit more readable, but I suppose there's no reason not to throw them all together. I'll do that.